apps/common/graphql/types.py

```diff
@@ -18,6 +18,6 @@ class UserResourceTypeMixin:
 
 @strawberry.type
 class ArchivableResourceTypeMixin:
-    is_archived: bool
+    is_archived: bool | None
```

Comment:
```suggestion
    is_archived: bool
```

Reply:
is_archived should be required for query but not required for mutation.
The default value comes from the database.